elements/howto-menu/howto-menu.js

```diff
@@ -0,0 +1,216 @@
+/*
+ * A menu is a widget that offers a list of choices to the user,
+ * such as a set of actions or functions. A menu is usually opened,
+ * or made visible, by activating a menu button, choosing an item in a menu
+ * that opens a sub menu, or by invoking a command, such as Shift + F10 in
+ * Windows, that opens a context specific menu.
+ *
+ * The element that opens the menu is referenced with aria-labelledby.
+ * TODO(devnook): Should it be aria-labelledby or aria-describedby?
```

Comment:
You could additionally add an `aria-describedby` field to help explain the purpose of the menu.

example:

label: "Options"
describedby: "Configuration for your personal twitter account"